packages/gatsby-transformer-xml/package.json

```diff
@@ -10,7 +10,7 @@
     "@babel/runtime": "^7.0.0",
     "bluebird": "^3.5.0",
     "lodash": "^4.17.10",
-    "xml-parser": "^1.2.1"
+    "xml-js": "^1.6.11"
   },
```

Comment:
you forget to change the plugin version number in [line 4](https://github.com/gatsbyjs/gatsby/pull/15598/files#diff-d9d0fb671b49b5f38c6211621468427fR4) from `2.1.0` to `3.0.0`

Reply:
Oh sure - I didn’t know if there was mechanics in place, somewhere in the pipeline, that would bump the package in the event of an accepted pull resurse containing breaking changes. Good to know, thanks :)

Reply:
i also don't know it exactly, but you mentioned `3.0` it in the `README`

Reply:
We always change the version number when running lerna (for publishing). So please don’t change the version numbers in this or future PRs - thanks!

Reply:
Thanks for clarifying :)